<ui version="4.0" >
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="minimumSize" >
   <size>
    <width>501</width>
    <height>309</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>MiKTeX Package Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="labelProgress1" >
       <property name="text" >
        <string>Downloading:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPackageName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar1" />
   </item>
   <item>
    <widget class="QLabel" name="labelProgress2" >
     <property name="text" >
      <string>Total</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar2" />
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Update statistics:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Removed files (packages)</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelRemovedFiles" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape" >
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" >
           <item>
            <widget class="QLabel" name="label_7" >
             <property name="text" >
              <string>Installed files (packages)</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelInstalledFiles" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape" >
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3" >
           <item>
            <widget class="QLabel" name="label_9" >
             <property name="text" >
              <string>Downloaded bytes:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDownloadedBytes" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape" >
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4" >
           <item>
            <widget class="QLabel" name="label_11" >
             <property name="text" >
              <string>KB/Sec</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelKbytesSec" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape" >
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
